sklearnex/tests/test_patching.py

```diff
@@ -328,3 +328,18 @@ def test_onedal_supported_member(name, member):
     patched = PATCHED_MODELS[name]
     sig = str(inspect.signature(getattr(patched, member)))
     assert "(self, method_name, *data)" == sig
+
+
+def test_check_entity_loaded():
+    # Test when sklearn is not loaded
+    assert check_entity_loaded(modules={}) is None
```

Comment:
add import for check_entity_loaded

Reply:
What do you mean exactly?

Reply:
CI fails because there is no import of `check_entity_loaded` at top of this file

Reply:
Sometimes the brain stops braining ...